kiagnose/internal/launcher/launcher.go

```
@@ -56,8 +56,11 @@ func (l Launcher) Run() (runErr error) {
 		}
 	}()
 
-	if err := l.checkup.Setup(); err != nil {
-		return err
+	setupErr := l.checkup.Setup()
+	_ = l.reporter.Report()
```

I do not really understand this one.
It is suppose to report the timestamp? If so, why it is doing this after the setup? makes sense to me to include it as well in the counting.

I also do not understand why the error is ignored.

---

The `startTimestamp` is set at the beginning of `Checkup.Setup()` to it is also being counted.

The error could be written to the log.